gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/Cluster.java

```diff
@@ -579,7 +657,9 @@ public Builder sslContext(final SslContext sslContext) {
          * File location for a SSL Certificate Chain to use when SSL is enabled. If this value is not provided and
          * SSL is enabled, the {@link TrustManager} will be established with a self-signed certificate which is NOT
          * suitable for production purposes.
+         * @deprecated
```

Comment:
Please include the full standard "deprecation message" wherever `@Deprecated` is used: 

`@deprecated As of release 3.2.10, replaced by blah blah blah"